File: sample/src/main/java/com/github/lany192/box/sample/data/binder/GirlBinder.java
package com.github.lany192.box.sample.data.binder;

import androidx.annotation.NonNull;

import com.github.lany192.arch.items.ItemBinder;
import com.github.lany192.box.sample.databinding.ItemGirlBinding;
import com.github.lany192.utils.ImageUtils;

public class GirlBinder extends ItemBinder<String, ItemGirlBinding> {

    @Override
    public void convert(@NonNull ItemGirlBinding binding, String url, int position) {
        ImageUtils.show(binding.image, url);
    }
}

File: sample/avatar/src/main/java/com/lany192/box/avatar/binder/GirlBinder.kt
package com.lany192.box.avatar.binder

import com.github.lany192.arch.items.ItemBinder
import com.github.lany192.utils.ImageUtils
import com.lany192.box.avatar.databinding.ItemGirlBinding

class GirlBinder : ItemBinder<String, ItemGirlBinding>() {
    override fun convert(binding: ItemGirlBinding, item: String, position: Int) {
        ImageUtils.show(binding.image, item)
    }
}
